<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensor interface</string>
  </property>
  <widget class="QTextBrowser" name="backgroundBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>197</y>
     <width>157</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="measureBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>254</y>
     <width>157</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="backgroundButton">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>197</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Background</string>
   </property>
  </widget>
  <widget class="QPushButton" name="measureButton">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>254</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Measure once</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recordButton">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>333</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start Continuous</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="filenameEdit">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>333</y>
     <width>133</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>filename.csv</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="portSelect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>127</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COM4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>69</y>
     <width>127</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="dialogueBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>280</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSubtract">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>234</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Subtract background</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="samplesBox">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>162</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>171</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Number of samples:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>108</y>
     <width>280</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>129</y>
     <width>283</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="PlotWidget" name="graphWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>357</y>
     <width>328</width>
     <height>145</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRecord">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>315</y>
     <width>127</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>285</y>
     <width>274</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSubtract_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>297</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Subtract background</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>303</y>
     <width>34</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>306</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot time (s) =</string>
   </property>
  </widget>
  <zorder>backgroundBrowser</zorder>
  <zorder>measureBrowser</zorder>
  <zorder>backgroundButton</zorder>
  <zorder>measureButton</zorder>
  <zorder>portSelect</zorder>
  <zorder>connectButton</zorder>
  <zorder>dialogueBox</zorder>
  <zorder>checkBoxSubtract</zorder>
  <zorder>samplesBox</zorder>
  <zorder>label</zorder>
  <zorder>line</zorder>
  <zorder>progressBar</zorder>
  <zorder>graphWidget</zorder>
  <zorder>checkBoxRecord</zorder>
  <zorder>line_3</zorder>
  <zorder>filenameEdit</zorder>
  <zorder>recordButton</zorder>
  <zorder>checkBoxSubtract_2</zorder>
  <zorder>spinBox</zorder>
  <zorder>label_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
